frontend/src/v5/ui/components/intercom/intercom.component.tsx

```diff
@@ -41,21 +41,29 @@ const IntercomButton = () => {
 	};
 
 	useEffect(() => {
-		if (isAuthenticated) {
+		if (isAuthenticated && currentUser.email) {
 			setNameAndMail();
 		} else {
-			shutdown();
+			hardShutdown();
 		}
-	}, [isAuthenticated]);
+	}, [isAuthenticated, currentUser.email]);
 
 	return <div />;
 };
 
 export const Intercom = () => {
 	const { intercomLicense } = clientConfigService;
+
+	let appId = intercomLicense;
+	const commentStartIndex = intercomLicense.indexOf('#');
```

Comment:
@Amantini1997 get rid of this hack, you should assume what's coming in is correct. It's only a problem in local (create an issue in local so @keifgwinn can deal with it)